### PR TITLE
[Docs] TCM 218943 weather station protocol

### PR DESCRIPTION
**Pull request by puuu**
As requested in #277, the documentation of the TCM protocol.

Comments are welcome.

### CODE REVIEW

File: docs/source/protocols/433.92/weather/tcm.rst [hunk 1]
```diff
@@ -0,0 +1,187 @@
+.. |yes| image:: ../../../images/yes.png
+.. |no| image:: ../../../images/no.png
+
+.. role:: underline
+   :class: underline
+
```
Comment (CurlyMoo) [inline docs/source/protocols/433.92/weather/tcm.rst]: Can you add a heading here like this:
```
TCM 218943
========
```

File: docs/source/protocols/433.92/weather/tcm.rst [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+.. rubric:: Supported Brands
+
++------------------------------+----------------+
+| **Brand**                    | **Protocol**   |
++------------------------------+----------------+
+| TCM 218943 weather stations  | tcm            |
++------------------------------+----------------+
+
+
```
Comment (CurlyMoo) [inline docs/source/protocols/433.92/weather/tcm.rst]: There are often bogus new lines in the source like here. Can you remove those?

**Comment by puuu** [comment]
OK, fixed it.

File: docs/source/protocols/433.92/weather/tcm.rst [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
++------------------+-------------+
+| Config           | |yes|       |
++------------------+-------------+
+
+.. rubric:: Supported Brands
+
++------------------------------+----------------+
+| **Brand**                    | **Protocol**   |
++------------------------------+----------------+
+| TCM 218943 weather stations  | tcm            |
```
Comment (CurlyMoo) [inline docs/source/protocols/433.92/weather/tcm.rst]: Can you remove the `weather stations` part here?

**Comment by puuu** [comment]
Done.

**Comment by CurlyMoo** [comment]
Thanks

**Pull request merged**